recipe/0001-include-pxd-pyx.patch

```diff
@@ -1,33 +0,0 @@
-From 240fbb78b2a7746a3ad026070cd5e5d6deaaa337 Mon Sep 17 00:00:00 2001
-From: Vladislav Zhurba <vzhurba@nvidia.com>
-Date: Fri, 8 Nov 2024 16:28:13 -0800
-Subject: [PATCH] Fix discovery for trampoline files
```

Comment:
Don't we need this patch to fix `cimport`s or is there another fix for that?

Reply:
What happened was that when we figured out the patch, I applied it directly to here ([Nov 10, 4:51pm ET](https://github.com/conda-forge/cuda-python-feedstock/pull/96/commits/288fdd427753cd9ea86eb6af0e4ee2401ce9a94e)) while upstream was still being updated (tagged at [Nov 12, 7:14pm ET](https://github.com/NVIDIA/cuda-python/releases/tag/v12.6.2.post1)). So at the time of rebuilding in this feedstock, we were using the previous tag + local patch. Now we are switching to the latest, fixed tag. The validity of the package (`cimport` is fixed) is ensured through the tests I added when rebuilding. Had I remembered the potential impact of versioneer I would have looked at `__version__` and decided to not rush into rebuilding...